openhtf/io/output/upload_json.py

```diff
@@ -0,0 +1,72 @@
+"""Module for uploading JSON-formatted test record files to server."""
+
+import os
+import logging
+import requests
+
+from json import JSONEncoder
+from openhtf import conf
+from openhtf.io.output.json_factory import OutputToJSON
+
+conf.Declare('project', 'project name')
```

Comment:
I wonder if 'path' would be more clear than 'target'. In fact, maybe we should take a step back and think about whether it even makes sense to break down the endpoint into parts rather than to just have an `http_upload_url` that encodes the host, port, and path. Thinking back, it seems like we're always taking a full URL for uploads and breaking it down by hand to put it into config anyway. Why not cut out the middleman? Open to discussion.